ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -121,6 +140,13 @@ class DemoListViewController: DemoTableViewController {
     let cellReuseIdentifier: String = "TableViewCell"
     private static var isFirstLaunch: Bool = true
     private static let lastDemoControllerKey: String = "LastDemoController"
+    private let cellTypeButton: UIButton = {
+        let button = Button()
+        button.setTitle("Type", for: .normal)
+        return button
+    }()
```

Comment:
What's the goal of adding this button? Is the title supposed to change to say the type?

Reply:
users can switch between the plain and insetgroup. it launches the menu

Reply:
Would it be possible to add it to the demo appearance controller instead? It seems odd to start adding customization options on the cell to launch the demo.

Reply:
once you create a tableview you can't edit the style. so it has to be done before the controller is created